Source/WebKit/UIProcess/Extensions/WebExtensionController.h

```diff
@@ -87,8 +87,10 @@ class WebExtensionController : public API::ObjectImpl<API::Object::Type::WebExte
     template<typename T, typename U>
     void sendToAllProcesses(const T& message, ObjectIdentifier<U> destinationID);
 
+#ifdef __OBJC__
     _WKWebExtensionController *wrapper() const { return (_WKWebExtensionController *)API::ObjectImpl<API::Object::Type::WebExtensionController>::wrapper(); }
```

Comment:
Would be nice to use `dynamic_objc_cast<_WKWebExtensionController>(...)` here from `<wtf/cocoa/TypeCastsCocoa.h>`.